docs/v2.4/install/migrate-from-legacy.md

```diff
@@ -1,6 +1,6 @@
 # Migrate from legacy Telepresence
 
-Telepresence (formerly referenced as Telepresence 2, which is the current major version) has different mechanics and requires a different mental model from [legacy Telepresence 1](/docs/v1/) when working with local instances of your services.
+Telepresence (formerly referenced as Telepresence 2, which is the current major version) has different mechanics and requires a different mental model from [legacy Telepresence 1](../../../v1/discussion/overview) when working with local instances of your services.
```

Comment:
I think we should keep the redirect since in prod that opens up a new tab, but if we have the redirect it does it in the same tab which I think is better UX.